### PR TITLE
Document .0 to unpack integer from Wrapping

### PR DESCRIPTION
**Pull request by sourcefrog**

### CODE REVIEW

File: src/libcore/num/mod.rs [hunk 1]
```diff
@@ -122,6 +122,9 @@ nonzero_integers! {
 /// all standard arithmetic operations on the underlying value are
 /// intended to have wrapping semantics.
 ///
+/// The enclosed integer can be retrieved through the `.0` index of the
+/// `Wrapping` tuple.
```
[the remark this answers is not in the document]
Comment (sourcefrog) [reply src/libcore/num/mod.rs]: Oh thanks, fixed.

**Review by frewsxcv — APPROVED** [review]

**Comment by frewsxcv** [comment]
Thanks!

@bors r+ rollup

**Comment by frewsxcv** [comment]
Good catch @mzji

**Comment by sourcefrog** [comment]
Sorry, I'm confused by git submodules, and that was not intentional.  I'll work out how to get them out.

**Comment by TimNN** [comment]
Ping from triage, @sourcefrog: Do you think you'll be able to make the requested changes? If you prefer, members of the Rust team can also push the requested updates to your branch. (Let me know if that is the case, and I can do it).

**Comment by sourcefrog** [comment]
@TimNN yes, doing that now.

I tried `git checkout` on the submodules but it seems more like I need to `rm -r` them to get git to forget about them. I'll push a new commit soon.

**Comment by sourcefrog** [comment]
OK should be clean now, please take another look.

**Pull request merged**